source/component/PasDoc_Parser.pas

```diff
@@ -2358,6 +2359,21 @@               { include "identifier = class of something;" as standard type }
             FreeAndNil(t);
             Exit;
           end;
+        KEY_TYPE: begin
+            t2 := PeekNextToken(LTemp);
+            IsTypeHelper := (t2.MyType = TOK_IDENTIFIER) and (t2.Info.StandardDirective = SD_HELPER);
+            if IsTypeHelper then
+            begin
```

Comment:
The test `(t2.MyType = TOK_IDENTIFIER) and (t2.Info.StandardDirective = SD_HELPER)` can be done easier, `T2.IsStandardDirective(SD_HELPER)`. And then it looks concise enough to not even need an extra local variable `IsTypeHelper ` IMHO, so you can just test 

```
if T2.IsStandardDirective(SD_HELPER) then
....
```

P.S. Call it `t2` not `T2`. I know we are not consistent in PasDoc with identifier naming, but the guideline is to follow `PascalCase` for everything, like in CGE ( https://castle-engine.io/coding_conventions#pascal_case ).

Reply:
Ok, applied the change.